indexer/src/main/java/org/uniprot/store/indexer/proteome/ProteomeConfig.java

```diff
@@ -45,12 +40,12 @@
     private String taxonomyFile;
 
     @Bean(name = "proteomeXmlReader")
-    public ItemReader<Proteome> proteomeReader() throws IOException {
+    public ItemReader<Proteome> proteomeReader() {
         return new ProteomeXmlEntryReader(proteomeXmlFilename);
     }
 
     @Bean(name = "proteomeXmlReader2")
```

Comment:
I know this reader was here before, but do you know what the difference between the two proteome readers are? It is probably a good idea if we could name the bean something a bit more clearly, showing the "intent" of it.

Reply:
This code will be refactored in my next task (to change proteome xml xsd). 
May I improve it as part of my next task?